assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -362,7 +362,7 @@ che.infra.kubernetes.pvc.storage_class_name=
 che.infra.kubernetes.pvc.quantity=10Gi
 
 # Pod that is launched when performing persistent volume claim maintenance jobs on OpenShift
-che.infra.kubernetes.pvc.jobs.image=centos:centos7
```

Comment:
It's still moving tag, but I think it's OK
CRW will fix it on its end as it currently does I think

Reply:
WIP to generate PRs at intervals to keep Che image refs up to date in https://github.com/eclipse/che/issues/18831